plugins/camel-tooling.vscode-wsdl2rest/0.0.7/meta.yaml

```diff
@@ -0,0 +1,14 @@
+id: camel-tooling.vscode-wsdl2rest
+version: 0.0.7
+type: VS Code extension
+name: wsdl2rest
+title: vscode-wsdl2rest
+description: A VS Code extension supporting wsdl2rest via Camel Rest DSL.
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/camel-tooling/vsextensions/vscode-wsdl2rest/0.0.7/vspackage
```

Comment:
could you please refer to the extension as GH release to avoid marketplace rate limit error - https://github.com/eclipse/che/issues/12840

Reply:
Will investigate further, thank you.

Reply:
currently the vsix are not pushed automatically to GitHub. The choice was made to not provide it to ensure a better tracking of usage through Marketplace.

Reply:
I cannot rebuild this version as it was based on staged dependencies which is no more available

`Failed to execute goal on project wsdl2rest-fatjar: Could not resolve dependencies for project org.github.camel.tooling.wsdl2rest:wsdl2rest-fatjar:jar:0.0.1-SNAPSHOT: Could not find artifact org.jboss.fuse.wsdl2rest:wsdl2rest-impl:jar:0.8.0.fuse-730090 in fuse-public-repository`

Reply:
I provided the artefact on the 0.0.7 release https://github.com/camel-tooling/vscode-wsdl2rest/releases/download/0.0.7/camel-tooling.vscode-wsdl2rest-0.0.7.vsix , expecting that it is a temporary hack